yield/tests/YieldSpec.hs

```diff
@@ -13,28 +13,28 @@ spec = describe "YieldT" do
   describe "Trivial examples" do
     it "" $ runYieldT noAggregation (pure 'x') `shouldReturn` ((), 'x')
     it "" $ runYieldT listAggregation (pure 'x') `shouldReturn` ("", 'x')
-    it "" $
-      runYieldT listAggregation (yield 'a' >> pure 'x') `shouldReturn` ("a", 'x')
-    it "" $
-      runYieldT noAggregation (yield 'a' >> pure 'x') `shouldReturn` ((), 'x')
-    it "" $
-      runYieldT listAggregation (yield 'a' >> yield 'b' >> pure 'c')
+    it ""
+      $ runYieldT listAggregation (yield 'a' >> pure 'x') `shouldReturn` ("a", 'x')
+    it ""
+      $ runYieldT noAggregation (yield 'a' >> pure 'x') `shouldReturn` ((), 'x')
+    it ""
+      $ runYieldT listAggregation (yield 'a' >> yield 'b' >> pure 'c')
         `shouldReturn` ("ab", 'c')
```

Comment:
FWIW, making all of these use a (redundant) `do` might be a good idea. That's our style because it doesn't result in a formatting change between one-line vs multi-line `it`s, `describe`s, etc.